### ISSUE TITLE
False-positive parameters align violation

### ISSUE THREAD

**Issue by ixti** [rubocop/rubocop#1017]
Subj on:

``` ruby
SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter[
  SimpleCov::Formatter::HTMLFormatter,
  Coveralls::SimpleCov::Formatter
]
```
